code/game/objects/effects/glowshroom.dm

```diff
@@ -1,6 +1,6 @@
-#define GLOWSHROOM_SPREAD_BASE_DIMINISH_FACTOR 10
-#define GLOWSHROOM_SPREAD_DIMINISH_FACTOR_PER_GLOWSHROOM 0.2
-#define GLOWSHROOM_BASE_INTEGRITY 60
```

Comment:
Not all defines need to be global, surely? Or am I just out of touch.

In Python, I would just put these as a module global constant, and leave them at that.

Reply:
honestly if we're going to have a botany file we should be putting them in there. I'm not necessarily against defines in files themselves, but I am against having a define file and then not using it